org.cockpit-project.certificates.metainfo.xml

```diff
@@ -2,15 +2,17 @@
 <component type="addon">
   <id>org.cockpit_project.certificates</id>
   <metadata_license>CC0-1.0</metadata_license>
-  <name>Starter Kit</name>
+  <name>Certificates</name>
   <summary>
-    Scaffolding for a cockpit module.
+    Manage your certificates with certmonger.
   </summary>
   <description>
     <p>
-      Scaffolding for a cockpit module.
+      You can request, view, adjust auto-renewal and delete certmonger certificates.
```

Comment:
Two missing commas, and "auto-renewal" is not a verb. How about "You can request, view, and maintain certmonger certificates."